HACKME_releases.md

```diff
@@ -0,0 +1,69 @@
+
+To find the list of commits, it is recommended that you use a tool that can provide some extra metadata to help with matching commits to pull requests. [changelog-maker](https://github.com/nodejs/changelog-maker) can help with this (requires Node.js be installed and the `npx` command be available):
+
+```
+npx changelog-maker --start-ref=v0.16.0  --reverse=true --find-matching-prs=true --md=true ipld go-ipld-prime
+```
+
+Alternatively, you can use `git log` and perform mapping to pull requests manually, e.g.:
+
+```
+git log --all --graph --date-order --abbrev-commit --decorate --oneline
+```
+
+*(where `--start-ref` points to name of the previous release tag)*
+
+### Curate and summarize
+
+The CHANGELOG should be informative for developers wanting to know what changes may pose a risk (highlight these!) and what changes introduce features they may be interested in using.
+
+1. Group commits to subsystem to create a two-level list. Subsections can include "Data Model", "Schemas", "Bindnode", "Selectors", "Codecs", and the meta-category of "Build" to describe changes local to the repository and not necessarily relevant to API consumers.
+2. If there are breaking, or potentially breaking changes, list them under a `### 🛠 Breaking Changes` section.
+3. Otherwise, prune the list of commits down to the set of changes relevant to users, and list them under a `### 🔦 Highlights` section.
+
+Note that there is also a **Planned/Upcoming Changes** section near the top of the CHANGELOG.md. Update this to remove _done_ items and add other items that may be nearing completion but not yet released.
+
+### Call-outs
+
+Add "special thanks" call-outs to individuals who have contributed meaningful changes to the release.
+
+## Propose a release
+
+After updating the CHANGELOG.md entry, also bump the version number appropriately in **version.json** file so the auto-release procedure can take care of tagging for you.
+
+Commit and propose the changes via a pull request to ipld/go-ipld-prime.
+
+## Release
+
+After a reasonable amount of time for feedback (usually at least a full global business day), the changes can be merged and a release tag will be created by the GitHub Actions.
+
+Use the GitHub UI to make a [release](https://github.com/ipld/go-ipld-prime/releases), copying in the contents of the CHANGELOG.md for that release.
+
+Drop in a note to the appropriate Matrix/Discord/Slack channel(s) for IPLD about the release.
+
+Optional: Protocol Labs staff can send an email to shipped@protocol.ai to describe the release, these are typically well-read and appreciated.
+
+## Checklist
+
+Prior to opening a release proposal pull request, create an issue with the following markdown checklist to help ensure the requisite steps are taken. The issue can also be used to alert subscribed developers to the timeframe and the approximate scope of changes in the release.
```

Comment:
Would it make sense to create a GH issue template for this?

Reply:
I don't really want to do this because we currently don't have any issue templates, so it introduces an extra step in the New Issue flow for everyone, just to support releases for one person.

If/when we start adding templates for other types of issues then we could add this one, otherwise it's going to be an extra "do you want to file an issue or make a release" in-between step for everyone.

Reply:
Yeah totally fair. Revisiting once there are other templates seems like a good idea. 😁